stellar-contract-env-host/src/host.rs

```diff
@@ -701,9 +777,30 @@ impl CheckedEnv for Host {
         todo!()
     }
 
+    #[cfg(feature = "vm")]
+    fn call3(
+        &self,
+        contract: Object,
+        func: RawVal,
+        a: RawVal,
+        b: RawVal,
+        c: RawVal,
+    ) -> Result<RawVal, HostError> {
+        self.call_n(
+            contract,
+            func,
+            &[
+                self.from_host_val(a)?,
+                self.from_host_val(b)?,
+                self.from_host_val(c)?,
+            ],
+        )
+    }
+
+    #[cfg(not(feature = "vm"))]
```

Comment:
FYI so you're not defining the function twice you can move the cfg inside the function for the contents.

Reply:
Thanks, did not know that. Fixed.